dali/dfu/dfurun.cpp

```diff
@@ -1226,13 +1227,16 @@ class CDFUengine: public CInterface, implements IDFUengine
                     if (!srcFile)
                         throw MakeStringException(-1,"Source file %s could not be found",tmp.str());
 
+                    srcName.set(tmp.str());
```

Comment:
picky: doesn't really need .str() here, the const char * operator, will automatically be invoked.

Reply:
Changed